assets/src/scss/layout/_footer.scss

```diff
@@ -30,137 +35,114 @@
   li {
     line-height: 1;
   }
-}
 
-.footer-links,
-.footer-links-secondary {
-  li {
-    line-height: 2rem;
-    @include medium-and-up {
-      display: inline-block;
-
-      &:after {
-        content: "\002F";
-        display: inline-block;
-        margin: 0 16px;
-      }
-
-      &:last-of-type:after {
-        display: none;
-      }
-    }
+  .container {
+    padding-bottom: 0;
+    display: flex;
+    flex-direction: column;
 
     @include large-and-up {
-      &:after {
-        margin: 0 24px 0;
-      }
+      flex-direction: row-reverse;
     }
   }
-}
 
-.footer-links {
-  --footer-menu-- {
-    color: $white;
-    font-weight: bold;
-    text-transform: uppercase;
-    margin-bottom: 16px;
-  }
+  .links {
+    margin-bottom: 24px;
+    width: 100%;
+    text-align: left;
+    column-count: 1;
 
-  li {
-    font-size: $font-size-md;
-  }
+    li {
+      font-size: $font-size-sm;
 
-  @include medium-and-up {
-    font-size: $font-size-sm;
-    margin-bottom: 16px;
-  }
+      &:not(:last-child) {
+        margin-bottom: 24px;
+      }
+    }
 
-  @include large-and-up {
-    font-size: $font-size-md;
-  }
-}
+    @include medium-and-up {
+      font-size: $font-size-md;
+      column-count: 3;
+    }
 
-.footer-links-secondary {
-  --footer-menu-secondary-- {
-    color: $grey-10;
-    text-transform: uppercase;
-    margin-bottom: 30px;
+    @include large-and-up {
+      column-count: 2;
+    }
   }
 
-  @include medium-and-up {
-    font-size: $font-size-xxs;
-  }
+  .copyright {
+    --footer--copyright-- {
```

Comment:
This is something we probably do want to change. These are some of the earlier variables I added, before there was the shorthand. If we use it here then the resulting variable name will be `--site-footer--copyright`, which is more specific because there may be many kinds of footers.

Reply:
Got it! It's already changed